### PR TITLE
Support tornado 6.2

### PR DESCRIPTION
**Pull request by fjetter**
Closes https://github.com/dask/distributed/issues/6669

Looks like all of the deprecations were already dealt with

### CODE REVIEW

File: continuous_integration/environment-3.10.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ dependencies:
   - sortedcollections
   - tblib
   - toolz
-  - tornado<6.2
+  - tornado >= 6.0.3
```
Comment (jrbourbeau) [inline continuous_integration/environment-3.10.yaml:41]:
```suggestion
  - tornado=6.2
```

**Review by jrbourbeau — COMMENTED** [review]
Thanks @fjetter! Unfortunately it looks like [`tornado=6.1` is getting pulled into CI](https://github.com/dask/distributed/runs/8156334161?check_suite_focus=true). Maybe there's some other dependency that's putting an upper bound on `tornado`? Might be worth pinning `tornado=6.2` in one of the environment files as a test

File: continuous_integration/environment-3.10.yaml [hunk 2]
```diff
@@ -38,7 +38,7 @@ dependencies:
   - sortedcollections
   - tblib
   - toolz
-  - tornado<6.2
+  - tornado=6.2
```
Comment (crusaderky) [inline continuous_integration/environment-3.10.yaml:41]: Otherwise, 6.1 is installed

**Comment by fjetter** [comment]
superseded by https://github.com/dask/distributed/pull/7286

**Pull request closed without merging**